src/zzz_od/gui/view/battle_assistant/battle_assistant_interface.py

```
@@ -147,12 +148,13 @@ def get_widget_at_top(self) -> QWidget:
         )
         top.add_widget(self.screenshot_opt)
 
-        self.gamepad_opt = ComboBoxSettingCard(
-            icon=FluentIcon.GAME, title='操作方式',
-            content='仅影响自动战斗。如需使用手柄，请先安装虚拟手柄依赖。',
-            options_enum=ControlMethodEnum,
+        self.background_mode_opt = SwitchSettingCard(
+            icon=FluentIcon.ROBOT,
+            title='后台模式（专享版）',
```

Ruff (0.15.21)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)